src/styles/main.scss

```diff
@@ -1,3 +1,35 @@
-body {
+@import "./utils/variables";
+
+@import "./blocks/header.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/card.scss";
+@import "./blocks/nav.scss";
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+p {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+
+  .main {
+    padding-top: 50px;
+
+    .container {
+      width: 944px;
+      display: flex;
+      justify-content: space-between;
+      flex-wrap: wrap;
+    }
+
+    .main__container {
+      margin: 0 auto;
+    }
```

Comment:
these selectors can't be elements of `html` make a separate block if you need them